src/layouts/product-integration-layout/components/header/style.module.css

```diff
@@ -1,46 +1,108 @@
-.header {
-	padding: 0;
-	background: var(--token-color-surface-faint);
-	max-width: 100%;
+.headerWrap {
+	@media (--dev-dot-tablet-up) {
+		display: grid;
+		grid-template-columns: auto 1fr;
+		grid-template-areas:
+			'icon content'
+			'icon badges';
+	}
+}
 
-	& .upperCard {
-		padding: 16px;
+.upper {
+	margin-bottom: 16px;
+	grid-area: content;
+
+	@media (--dev-dot-tablet-up) {
 		display: flex;
+		flex-direction: row-reverse;
```

Comment:
I think this `row-reverse`, paired with the fact that the version selector is hidden for `<= 1` versions, results in the following situation:

(Acknowledging that this would've been hard to catch, given the staging data which had multiple versions)

<img width="749" alt="CleanShot 2023-01-30 at 17 56 15@2x" src="https://user-images.githubusercontent.com/26389321/215615311-f5f22037-5121-4631-be77-d45dd40535d1.png">